Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:10-alpine
+
+LABEL maintainer "TEAM BAP <engineering@bureauxapartager.com>"
```

Comment:
I'm not sure it makes sense to add a Dockerfile to the main repo, since different parties are likely to use it in slightly different ways, use a different `FROM` image, etc. Is tehre a way to accomplish what you need without adding this here?